docs/docs/external/getting-started/test-network/setup-local.md

```diff
@@ -69,17 +55,41 @@ Before starting, make sure you have the following software installed on your hos
 ### Credentials
 Make sure you have an SSH or GPG key registered in https://github.com to allow seamless cloning of repositories (at present, various setup scripts clone repositories using the `https://` prefix but this may change to `git@` in the future).
 
-#### Package Access Token:
-Create a personal access token with `read:packages` access in github in order to use modules published in github packages. Refer [Creating a Personal Access Token](https://docs.github.com/en/github/authenticating-to-github/keeping-your-account-and-data-secure/creating-a-personal-access-token) for help.
-
 ## Getting the Code and Documentation
 
 Clone the [weaver-dlt-interoperability](https://github.com/hyperledger-labs/weaver-dlt-interoperability) repository. The code to get a basic test network up and running and test data-sharing interoperation flows lies in the subfolder `tests/network-setups`, which should be your starting point, though the setups will rely on other parts of the repository, as you will find out in the instructions given on this page.
 
+## Common Structures
+
+The `common/protos` folder contains structure definitions in the protobuf format that are used by all the different components. The various `common/protos-*` folders are meant to contain compiled protobufs (in different languages).
+
+To compile the protobufs, do the following:
+- Navigate to the `common/protos-js` folder.
+- Run the following command:
+  ```bash
+  make build
+  ```
+- Navigate to the `common/protos-go` folder.
+- Run the following command:
+  ```bash
+  make build
+  ```
```

Comment:
`common/protos-java-kt` build is missing.
Also building `protos-go` isn't required actually, because we push the built/compiled protos-go into the repo.

Reply:
Added `protos-java-kt` instruction. Retained `protos-go` instruction because we want to support user modifications to `protos` before building.